Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp

```diff
@@ -245,15 +234,17 @@ namespace AssetProcessor
         return params;
     }
 
-    AZStd::unique_ptr<AzFramework::ProcessWatcher> Builder::LaunchProcess(const char* fullExePath, const AZStd::string& params) const
+    AZStd::unique_ptr<AzFramework::ProcessWatcher> Builder::LaunchProcess(const char* fullExePath, const AZStd::vector<AZStd::string>& params) const
     {
         AzFramework::ProcessLauncher::ProcessLaunchInfo processLaunchInfo;
         processLaunchInfo.m_processExecutableString = fullExePath;
-        processLaunchInfo.m_commandlineParameters = AZStd::string::format("\"%s\" %s", fullExePath, params.c_str());
+        processLaunchInfo.m_commandlineParameters = AZStd::vector<AZStd::string>{ fullExePath };
+        auto& commandLineArray = AZStd::get<AZStd::vector<AZStd::string>>(processLaunchInfo.m_commandlineParameters);
+        commandLineArray.insert(commandLineArray.end(), params.begin(), params.end());
```

Comment:
Actually this can be to avoid the call to `AZstd;:get` to look up the alternative for the variant.

```suggestion
        AZStd::vector<AZStd::string> commandLineArray{ fullExePath };
        commandLineArray.insert(commandLineArray.end(), params.begin(), params.end());
        processLaunchInfo.m_commandlineParameters = AZStd::move(commandLineArray);
```